task3.3/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace task3._3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            double x;
+            Console.WriteLine("enter x:");
+            x = double.Parse(Console.ReadLine());
+            Console.WriteLine("y = " + Funk(x));
+        }
+        static double Funk(double x)
+        {
+            return Math.Pow(Math.E, (1 - (x * x)) / (1 + (x * x)));
```

Comment:
В классе Math есть отдельная функция для вычисления экспоненты (степени числа e). Лучше использовать её).
лишние скобки затрудняют чтение кода. Помните о приоритетах операций!
```suggestion
            return Math.Exp((1 - x * x) / (1 + x * x));
```